Packs/Wordpress/Integrations/Wordpress/Wordpress.yml

```diff
@@ -0,0 +1,2446 @@
+      name: slug
+      required: false
+      secret: false
+    - description: 'Limit result set to users matching at least one specific role provided (CSV).'
+      isArray: false
+      name: roles
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      description: 'Limit result set to users who are considered authors.'
+      isArray: false
+      name: who
+      predefined:
+      - "authors"
+      required: false
+      secret: false
+    outputs:
+    - contextPath: Wordpress.Users.meta
+      description: 'Metadata'
+      type: List
+    - contextPath: Wordpress.Users.avatar_urls
+      description: 'Avatar URLs'
+      type: List
+    - contextPath: Wordpress.Users.name
+      description: 'Name'
+      type: String
+    - contextPath: Wordpress.Users.slug
+      description: 'Slug'
+      type: String
+    - contextPath: Wordpress.Users.url
+      description: 'URL'
+      type: String
+    - contextPath: Wordpress.Users._links
+      description: 'Links'
+      type: List
+    - contextPath: Wordpress.Users.link
+      description: 'Link'
+      type: String
+    - contextPath: Wordpress.Users.id
+      description: 'User ID'
+      type: Number
+    - contextPath: Wordpress.Users.description
+      description: 'Description'
+      type: String
+    deprecated: false
+    description: 'List users'
+    execution: false
+  - name: wordpress-get-user
+    arguments:
+    - default: false
+      description: 'User ID.'
+      isArray: false
+      name: id
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      defaultValue: "view"
+      description: 'Scope under which the request is made; determines fields present in response (default is view).'
+      isArray: false
+      name: context
+      required: false
+      secret: false
+      predefined:
+      - "view"
+      - "edit"
+      - "embed"
+    deprecated: false
+    description: 'Retrieve a user.'
+    execution: false
+    outputs:
+    - contextPath: Wordpress.Users.meta
+      description: 'Metadata'
+      type: List
+    - contextPath: Wordpress.Users.avatar_urls
+      description: 'Avatar URLs'
+      type: List
+    - contextPath: Wordpress.Users.name
+      description: 'Name'
+      type: String
+    - contextPath: Wordpress.Users.slug
+      description: 'Slug'
+      type: String
+    - contextPath: Wordpress.Users.url
+      description: 'URL'
+      type: String
+    - contextPath: Wordpress.Users._links
+      description: 'Links'
+      type: List
+    - contextPath: Wordpress.Users.link
+      description: 'Link'
+      type: String
+    - contextPath: Wordpress.Users.id
+      description: 'User ID'
+      type: Number
+    - contextPath: Wordpress.Users.description
+      description: 'Description'
+      type: String
+  - name: wordpress-create-user
+    arguments:
+    - defaultValue: "en_US"
+      description: 'Locale for the user (default is en_US).'
+      isArray: false
+      name: status
```

Comment:
why the name aren't `locale`?

Reply:
Well spotted. Too many copy and pastes :)